packages/helm-charts/common/templates/_helpers.tpl

```diff
@@ -535,3 +532,21 @@ prometheus.io/port: "{{ $pprof.port | default 6060 }}"
   - name: data
     mountPath: /root/.celo
 {{- end -}}
+
+{{- define "common.geth-add-metrics-pprof-config" -}}
+{{- if .metrics | default true }}
+ADDITIONAL_FLAGS="${ADDITIONAL_FLAGS} --metrics"
+{{- end }}
+{{- if (or .Values.metrics .Values.pprof.enabled) | default false }}
+# Check the format of pprof cmd arguments
+set +e
+geth --help | grep 'pprof.port' >/dev/null
+pprof_new_format=$?
+set -e
+if [ $pprof_new_format -eq 0 ]; then
+  ADDITIONAL_FLAGS="${ADDITIONAL_FLAGS} --pprof --pprof.port {{ .Values.pprof.port | default "6060" }} --pprof.addr 0.0.0.0"
+else
+  ADDITIONAL_FLAGS="${ADDITIONAL_FLAGS} --pprof --pprofport {{ .Values.pprof.port | default "6060" }} --pprofaddr 0.0.0.0"
```

Comment:
I feel like we may have supported both options for a while and may not need to have this if statement. Let me double check which version was the first to support this.

Reply:
I looked, and this is supported after v1.4. We probably have some code on 1.3, but after espresso, we should be able to remove this.

Reply:
The rpc/ws/http flagas has been also updated. We can keep this for now, and drop compatibility with <1.5 after the hardfork activation.